### PR TITLE
Promote WindowsGMSA feature to GA.

### PR DESCRIPTION
**Pull request by ddebroy**
Doc update for promotion of GMSA feature to Stable

### CODE REVIEW

**Comment by sftim** [comment]
I'll tell Prow this is a placeholder
/retitle [WIP] Promote WindowsGMSA feature to GA

**Comment by irvifa** [comment]
Hi @ddebroy 👋 any plan on adding more docs on this part or is this already done?

**Comment by JeremyWx** [comment]
@irvifa the docs for this feature are complete.

**Comment by irvifa** [comment]
@JeremyWx  thanks for the confirmation. cc @ddebroy 

@sftim do we need additional tech reviewer for this? This is already complete

**Comment by sftim** [comment]
One thing though @ddebroy - can you rebase this against _dev-1.18_?

**Comment by VineethReddy02** [comment]
👋 Please rebase this PR on `dev-1.18` in order to avoid merge conflicts.

Feel free to `/hold cancel` after you've rebased.

/hold

**Comment by sftim** [comment]
@ddebroy please also update the PR description if this is no longer a placeholder PR.

**Comment by JeremyWx** [comment]
@sftim @ddebroy will be on leave for at least the next few weeks.  He has given me access to his repo.  To be clear, I will need to rebase and update the description for this to pass?

**Comment by sftim** [comment]
Ah, it's OK. An updated description makes reviewers more likely to see this, but it's fine.
Can you do the rebase though (and force-push it) @JeremyWx?

**Comment by JeremyWx** [comment]
@sftim yes I will get that done today, tomorrow at the latest.

**Comment by kbhawkey** [comment]
@JeremyWx @ddebroy , Is this pull request a duplicate of https://github.com/kubernetes/website/pull/19604 (close if a duplicate)?

**Comment by irvifa** [comment]
Hi @JeremyWx @ddebroy 
We are very close to the docs final deadline i.e All 1.18 enhancements docs should be in ready to merge state by Monday 16th March. Please make sure to complete all the docs reviews before the deadline.

Thanks!

**Comment by VineethReddy02** [comment]
Hi @ddebroy 
Today is the deadline for 1.18 docs to be ready to merge state. Can we know the status of this PR?

**Comment by ddebroy** [comment]
Rebased and updated the PR.

**Comment by ddebroy** [comment]
@sftim this is now ready.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -277,6 +277,7 @@ different Kubernetes components.
 | `WatchBookmark` | `false` | Alpha | 1.15 | 1.15 |
 | `WatchBookmark` | `true` | Beta | 1.16 | 1.16 |
 | `WatchBookmark` | `true` | GA | 1.17 | - |
+| `WindowsGMSA` | `true` | GA | 1.18 | - |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Was this feature previously alpha / beta? If so, list those stages as well.

Comment (ddebroy) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Done @sftim

**Pull request merged**